public/js/pimcore/document/editables/video.js

```
@@ -20,7 +20,6 @@ pimcore.document.editables.video = Class.create(pimcore.document.editable, {
     initialize: function($super, id, name, config, data, inherited) {
         $super(id, name, config, data, inherited);
 
-        data.allowedTypes = config.allowedTypes;
```

This was just added in #747. We should probably take another look at this ticket as well.

---

@roland4432 @jdreesen The allowedTypes are already set from the config on the data in the editable php file. See https://github.com/pimcore/pimcore/blob/38728393ce02dfaa532f54c4fe99731b26638747/models/Document/Editable/Video.php#L1001

---

@roland4432 What was your `allowedTypes` configuration? Could it be that you used a wrong type in your array? Only an array with the values "asset", "youtube", "vimeo", and "dailymotion" is allowed. Otherwise, the configuration will be ignored. I think we should throw an exception when an incorrect type is used, rather than silently ignoring the configuration.

---

> I think we should throw an exception when an incorrect type is used, rather than silently ignoring the configuration.

:+1: